src/status_im/log_level/core.cljs

```diff
@@ -13,9 +13,9 @@
                         (assoc settings :log-level log-level)
                         (dissoc settings :log-level))]
     (fx/merge cofx
-              (multiaccounts.update/update-settings new-settings
-                                                    {})
-              (node/prepare-new-config {:on-success #(when (not= (node/get-log-level settings) (node/get-log-level new-settings))
+              (multiaccounts.update/update-settings new-settings {})
+              (node/prepare-new-config {:on-success #(when (not= (node/get-log-level settings)
+                                                                 (node/get-log-level new-settings))
```

Comment:
Are there any changes at all? Is it just code formatting?

Reply:
just formatted to be the same layout as in `fleet.core` and more readable